src/Spago/Config.purs

```diff
@@ -448,10 +448,15 @@ rootPackageToWorkspacePackage { rootPackage, workspaceDoc } = do
 
 workspacePackageToLockfilePackage :: WorkspacePackage -> Tuple PackageName Lock.WorkspaceLockPackage
 workspacePackageToLockfilePackage { path, package } = Tuple package.name
-  { path
+  { path: forwardSlashPath
   , core: { dependencies: package.dependencies, build_plan: mempty }
   , test: { dependencies: foldMap _.dependencies package.test, build_plan: mempty }
   }
+  where
+  -- When writing lockfile, we specifically always use forward slash as path
+  -- separator to ensure that the lockfile doesn't keep changing when team
+  -- members run Spago on different platforms.
+  forwardSlashPath = path # String.replaceAll (String.Pattern Path.sep) (String.Replacement "/")
```

Comment:
We have `withForwardSlashes` in the Prelude

Reply:
Oh indeed! Haven't noticed that.